src/coreclr/ilasm/sha256.h

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//*****************************************************************************
+// sha256.h
+//
+
+//
+// contains implementation of sha256 hash algorithm
+//
+//*****************************************************************************
+#ifndef __sha256__h__
+#define __sha256__h__
```

Comment:
nit: `__` names are reserved by convention.
```suggestion
#ifndef HAVE_ILASM_SHA256_H
#define HAVE_ILASM_SHA256_H
```